src/StorageFlattening.cpp

```diff
@@ -341,6 +413,9 @@ Stmt storage_flattening(Stmt s,
                         const vector<Function> &outputs,
                         const map<string, Function> &env,
                         const Target &target) {
+    // The OpenGL backend requires loop mins to be zero'd at this point.
```

Comment:
They need to be zeroed at some point. This was previously done inside the pass I deleted, and doing it here keeps it at the equivalent point in lowering.